src/DataMessenger/DataMessenger.js

```diff
@@ -485,14 +485,14 @@ export function DataMessenger(options = {}) {
             </div>
         `);
         const button = htmlToElement(`
-            <button class="autoql-vanilla-chata-btn primary"
+            <button class="autoql-vanilla-chata-btn autoql-vanilla-primary"
             style="padding: 5px 16px; margin: 10px 5px 2px;">
                 Create a New Notification
             </button>
         `);
 
         button.onclick = () => {
-            var modalView = new NotificationSettingsModal(obj.options);
+           /*  var modalView = new NotificationSettingsModal(obj.options);
```

Comment:
Could you please delete the comment or leave an explanation such as 
//TO DO --- (Example: why this comment must be kept here? Maybe you want to uncomment those for later use, some reason like that)